testing_and_setup/compass/ocean/hurricane/interpolate_time_varying_forcing.py

```diff
@@ -12,6 +12,7 @@
 import cartopy.crs as ccrs
 import cartopy.feature as cfeature
 from scipy import interpolate
+import write_forcing_file
```

Comment:
This will affect more than just this code here, correct?  May have to double-check to make sure that two files aren't needed when using `interpolate_time_varying_forcing.py` in other pre-existing scripts.

Reply:
This gets imported from the target directory of the linked script.